Client/src/Components/DashboardMenu/index.jsx

```diff
@@ -6,6 +6,8 @@ import Incidents from "../../assets/Images/Icon-warning-gray.png";
 import SensorsIcon from "../../assets/Images/Icon-signal-gray.png";
 import AllInclusiveIcon from "../../assets/Images/Icon-link-gray.png";
 import SettingsIcon from "../../assets/Images/Icon-setting-gray.png";
+import React, { useState, useEffect } from "react";
+import { useNavigate, useLocation } from "react-router-dom";
 
```

Comment:
Do we want to be using redux to handle the navigation state as well?  @veyselboybay

Reply:
It'd be nice to have a central navigation state however, let's skip it for now. We can come back to it later.